lib/std/http/WebSocket.zig

```diff
@@ -23,6 +23,9 @@ pub fn init(
     send_buffer: []u8,
     recv_buffer: []align(4) u8,
 ) InitError!bool {
+    if (request.head.version == .@"HTTP/1.1" and request.head.method != .GET)
```

Comment:
```suggestion
    if (request.head.version != .@"HTTP/1.1" or request.head.method != .GET)
```

Unless I'm misreading the RFC?